### ISSUE TITLE
 A data scientist is trying to determine how a model is doing based on training evaluation. The train accuracy plateaus out at around 70% and the validation accuracy is 67%. How should the data scientist interpret these results?

### ISSUE THREAD

**Issue by epassaro** [epassaro/aws-mls-c01#17]
### A data scientist is trying to determine how a model is doing based on training evaluation. The train accuracy plateaus out at around 70% and the validation accuracy is 67%. How should the data scientist interpret these results?

- [ ] The model is underfit and needs more complexity
- [ ] The model is overfit and needs less complexity
- [ ] The model is generalizing well and isn't overfit or underfit
- [ ] The model is overfit and underfit and needs more epochs

<details>
<summary><h3>Solución</h3></summary>

- [x] The model is underfit and needs more complexity
- [ ] The model is overfit and needs less complexity
- [ ] The model is generalizing well and isn't overfit or underfit
- [ ] The model is overfit and underfit and needs more epochs

</details>

<details>
<summary><h3>Razonamiento</h3></summary>
 

- La respuesta B es claramente incorrecta ya que tiene poca _accuracy_ en el conjunto de prueba
- En la respuesta D termino de entender la primera parte, aunque como el enunciado dice que la curva tiene un _plateau_ entonces darle más épocas no va a solucionar nada 
- La respuesta correcta es la A ya que el resultado es bastante pobre y el _training set_ no se separa mucho del _test set_
</details>
